src/compare-tree/key-path.js

```diff
@@ -0,0 +1,40 @@
+const KEY_DELIMETER  = '.'
+const PATH_DELIMETER = ','
+
+import {
+  castPath
+} from '../utils'
+
+function byKeyLength (a, b) {
+  return a.length > b.length ? 1 : -1
+}
+
+export function getKeyString (value) {
+  return value.join(KEY_DELIMETER)
+}
+
+export function getKeyPaths (value) {
+  if (value == null) {
+    return []
+  }
+
+  let paths = value
+
+  if (Array.isArray(value) === false) {
+    paths = `${paths}`.split(PATH_DELIMETER)
+  }
+
+  return paths.map(castPath)
```

Comment:
`PATH_DELIMETER` is a comma. 

This function turns a list of key paths, like `users,query.selected` into `[ ['users'], ['query', 'selected' ] ]`

That way, you can set up a subscription that listens to both key paths at once, executing a callback with the values of both as parameters, like `function (users, selected) {}`